plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/order-summary/style.scss

```
@@ -1,4 +1,7 @@
 .wc-block-components-order-summary {
+	// compensate for removing Panel.
```

```suggestion
	// Compensate for removing Panel.
```
Nit, but comments should start with a capital letter